libmamba/ext/solv-cpp/tests/CMakeLists.txt

```diff
@@ -19,7 +20,8 @@ add_executable(
     src/test_transaction.cpp
 )
 target_include_directories(test_solv_cpp PRIVATE src/)
-target_compile_definitions(test_solv_cpp PRIVATE DOCTEST_CONFIG_USE_STD_HEADERS)
-
-find_package(doctest REQUIRED)
-target_link_libraries(test_solv_cpp PRIVATE doctest::doctest solv::cpp)
+find_package(Catch2 REQUIRED)
+target_link_libraries(test_solv_cpp PRIVATE Catch2::Catch2WithMain solv::cpp)
+set_target_properties(
+    test_solv_cpp PROPERTIES COMPILE_DEFINITIONS CATCH_CONFIG_ENABLE_ALL_STRINGMAKERS
```

Comment:
This allows us to enable nice output for many std types without having to write implementations ourselves (this is from Catch's source code):

```
#if defined(CATCH_CONFIG_ENABLE_ALL_STRINGMAKERS)
#  define CATCH_CONFIG_ENABLE_PAIR_STRINGMAKER
#  define CATCH_CONFIG_ENABLE_TUPLE_STRINGMAKER
#  define CATCH_CONFIG_ENABLE_VARIANT_STRINGMAKER
#  define CATCH_CONFIG_ENABLE_OPTIONAL_STRINGMAKER
#endif
```